app/views/course/achievements/index.html.slim

```diff
@@ -4,10 +4,11 @@ table.table.achievement-list.table-hover
   thead
     tr
       th
+      th = t('.title')
       th = t('.description')
       th = t('.requirements')
       - if can?(:manage, Course::Achievement)
-      th = t('.published')
+        th = t('.published')
```

Comment:
@allenwq @dariusf how are we dealing with published/unpublished items? Column, icon, what?